kythe/typescript/indexer.ts

```diff
@@ -1863,6 +1852,8 @@ export function index(
       }
     }
   }
+
+  return diags;
```

Comment:
Is anything done with these diagnostics? Like logging or emitting kythe diagnostic nodes?

Reply:
Not here.  We can in the consumer of this API if we want.  I'm not sure it's useful for us to log anything, because if the compile fails we should already know about it via other means.

Reply:
As long as we know about it somehow for when we are triaging a bug report, that sounds good to me.